addons/jenkins/1.x/jenkins-2.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: jenkins
+  labels:
+    kubeaddons.mesosphere.io/name: jenkins
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "2.6.4-1"
+    appversion.kubeaddons.mesosphere.io/jenkins: "2.6.4"
+    values.chart.helm.kubeaddons.mesosphere.io/jenkins: "https://raw.githubusercontent.com/jenkinsci/helm-charts/2e646e13bbc69f904d8d38f73249b395ff38e8b3/charts/jenkins/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.16.3
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
```

Comment:
Looks like GCP is missing from the list. We can add that to maintain uniformity across other addons.

Reply:
@sebbrandt87 is vsphere a thing, yet?

Reply:
@makkes yes `vsphere` is a thing yet, but not released yet.